local-running/compose.yaml

```diff
@@ -313,3 +313,26 @@ services:
       - "5012:5012"
     volumes:
       - "../../ipv-config/stubs/di-ipv-credential-issuer-stub:/etc/client-configs:ro"
+
+  bav-stub:
```

Comment:
I'm wondering if using `bav` as the id for this CRI is right. My understanding is that the CRI will return an evidence type CRI, not verification. Maybe we should change to from `bav` to something like `bankAccount`. I realise this will need updating in our config too..

Reply:
You're right it's evidence not verification so the name is annoying - but 'Bank Account Verification CRI' is the official name https://github.com/govuk-one-login/architecture/blob/b15fedc4e4ae1f924092ce0272eeb95a38eb2856/rfc/0059-bank-account-verification.md I think because it uses the HMRC banking 'verification' API

Reply:
Urgh. Fair enough.